package.json

```diff
@@ -32,6 +32,7 @@
     "mocha": "^3.0.0",
     "mocha-eslint": "^3.0.1",
     "quick-temp": "^0.1.3",
+    "sinon": "^1.17.7",
```

Comment:
I would like to understand the disdain for `sinon`, as I find it much cleaner than manually stubbing out a function and then having to assert on a variable or something.

That said, I'll go ahead and update these tests to remove `sinon`, but threading `console` through would be far beyond the scope of this PR.